pandas/core/algorithms.py

```diff
@@ -482,9 +482,11 @@ def isin(comps: AnyArrayLike, values: AnyArrayLike) -> np.ndarray:
     comps = _ensure_arraylike(comps)
     comps = extract_array(comps, extract_numpy=True)
     if is_extension_array_dtype(comps.dtype):
```

Comment:
just make this an `isinstance(comps, ExtensionArray):`

Reply:
Can't do that.  Creates a circular import.

Reply:
could do `if not isinstance(comps, np.ndarray)`.  after the `extract_array` call comps should be ArrayLike.

Reply:
True.  But I think I will do that in another PR related to fixing args for `isin`